.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 /composer.lock
 /phpunit.xml
+.idea
```

Comment:
The **.gitignore** file will not contain every IDE's cache folders, please add to your global .gitignore